generator/src/components/getting_started_specific/LoadFile.js

```diff
@@ -1,17 +1,44 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { loadConfiguration } from '../../redux/actions'
+import { navigate } from 'gatsby'
 
 //TODO: Complete functions for uploading files 
 export default function LoadFile() {
+    const dispatch = useDispatch()
+    const [selectedFile, setSelectedFile] = useState()
+    const [isFileSelected, setIsFileSelected] = useState(false)
```

Comment:
is this needed?

Reply:
added in missing code to prevent uploads with no file selected